packedForest/src/forestTypes/unsupervisedForests/stratifiedInNodeClassIndicesUnsupervised.h

```diff
@@ -164,7 +176,7 @@ namespace fp{
 			}
 
 			inline void initializeBinnedSamples(){
-				if(useBin()){
+				/*if(useBin()){
```

Comment:
styling -- can we avoid block comments in general?  it makes it difficult for git history to show changes on a particular line

in this case -- perhaps just delete the code block?